opencensus/stats/testing/test_utils.h

```diff
@@ -31,11 +31,18 @@ namespace testing {
 
 class TestUtils final {
  public:
+  // Makes a ViewData, using absl::UnixEpoch() as the start time.
   static ViewData MakeViewData(
       const ViewDescriptor& descriptor,
       std::initializer_list<std::pair<std::vector<std::string>, double>>
           values);
 
+  // Makes a ViewData, using the specified times for each timeseries.
+  static ViewData MakeViewData(
+      const ViewDescriptor& descriptor,
+      std::vector<std::tuple<std::vector<std::string>, double, absl::Time>>
```

Comment:
Not initializer_list like above?

Reply:
I wasn't able to get the overloading to work with std::initializer, so I have just given all of these methods different names. See:
(https://travis-ci.org/github/census-instrumentation/opencensus-cpp/jobs/688632251)

What I have done now is have (1) an private method which accepts the vector, which the public methods which accept the initializer lists. (2) Avoided overloading altogether and given the methods different names

LMK what you think. :)

Reply:
Fair enough. :)